### PR TITLE
WIP

### PR DESCRIPTION
**Pull request by cgoncas**

### CODE REVIEW

**Comment by cgoncas** [comment]
Continue in https://github.com/cgoncas/liferay-portal/pull/489

**Comment by dgarciasarai** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by dgarciasarai** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/dao/search/ContentDashboardInfoItemDisplaySearchContainerFactory.java [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+
+	public SearchContainer<ContentDashboardInfoItem<?>> create()
+		throws PortletException {
+
+		PortletURL portletURL = PortletURLUtil.clone(
+			_renderResponse.createRenderURL(), _renderResponse);
+
+		SearchContainer<ContentDashboardInfoItem<?>> searchContainer =
+			new SearchContainer<>(
+				_renderRequest, portletURL, null, "there-are-no-contents");
```
Comment (cgoncas) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/dao/search/ContentDashboardInfoItemDisplaySearchContainerFactory.java:89]: @dgarciasarai , así inicializamos el mensaje

**Pull request closed without merging**